_episodes/03-matplotlib.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Visualizing Tabular Data
+teaching: 30
+exercises: 20
+questions:
+- "How can I visualize tabular data in Python?"
+- "How can I group several plots together?"
+objectives:
+- "Plot simple graphs from data."
+- "Group several graphs in a single figure."
+keypoints:
+- "Use the `pyplot` library from `matplotlib` for creating simple visualizations."
```

Comment:
below we say "pyplot module from matplotlib library". Maybe we could use the same here. What do you think?

Reply:
Thanks, Anne. I think it makes sense to use the same wording.